en/docs/learn/configuring-password-patterns.md

```diff
@@ -0,0 +1,92 @@
+# Configuring Password Patterns
+
+WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. 
+
+<img src="../../assets/img/learn/password-pattern.png" alt="Password Pattern" width="700">  
+
+## Scenario
+
+**Pickup** is a cab company that has many employees who use different credentials to sign in to different internal enterprise applications. Robert who is an administrator at Pickup wants enforce the following password pattern when logging in.
+
```

Comment:
Robert{comma} who is an administrator at Pickup {comma} wants **to** enforce the following password pattern when logging in.

Reply:
Fixed